src/app/config/config.c

```diff
@@ -765,16 +765,20 @@ static const config_var_t option_vars_[] = {
 
 /** List of default directory authorities */
 static const char *default_authorities[] = {
+#ifndef COCCI
 #include "auth_dirs.inc"
+#endif
   NULL
 };
 
 /** List of fallback directory authorities. The list is generated by opt-in of
  * relays that meet certain stability criteria.
  */
 static const char *default_fallbacks[] = {
+#ifndef COCCI
 #include "fallback_dirs.inc"
-  NULL
+#endif
+   NULL
```

Comment:
Spacing: why is this NULL indented 3 spaces?

Reply:
2df7442afd0d19377115f75d8a4b120092ba2a19 is the fixup.